### PR TITLE
Liveplot

### PR DESCRIPTION
**Pull request by licode**
Add initial guess to livefitplot. @danielballan

### CODE REVIEW

File: bluesky/callbacks/core.py [hunk 1]
```diff
@@ -856,8 +856,10 @@ def start(self, doc):
         self.livefit.start(doc)
         self.x, = self.livefit.independent_vars.keys()  # in case it changed
         super().start(doc)
+        self.init_guess_line, = self.ax.plot([], [], color='grey', label='init guess')
+        self.lines.append(self.init_guess_line)
         # Put fit above other lines (default 2) but below text (default 3).
-        self.current_line.set_zorder(2.5)
+        [line.set_zorder(2.5) for line in self.current_line]
```
Comment (licode) [inline bluesky/callbacks/core.py]: should be self.lines.

**Comment by danielballan** [comment]
Looks good. One last request: would you run an example (say, [this one](https://nsls-ii.github.io/bluesky/callbacks.html#livefitplot)) locally and post a screenshot or saved PNG of the result so we can all see what it looks like now?

**Comment by licode** [comment]
Sure, this is a good way to test.

**Comment by licode** [comment]
```
from bluesky.callbacks import LiveFitPlot
lpf = LiveFitPlot(lf, color='r')

RE(scan([noisy_det], motor, -1, 1, 100), lpf)
RE(scan([noisy_det], motor, -1, 2, 100), lpf)
```
screen shot 2017-10-18 at 12 07 19 pm

**Comment by danielballan** [comment]
Awesome! This is a great improvement. Thanks, @licode.

**Pull request merged**